content/en/docs/guides/deadlines.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deadlines
+description: >-
+  Explains how deadlines can be used to effectively deal with unreliable
+  backends.
+---
+
+### Overview
+
+A deadline is used to specify how long a client is willing to wait for a server
```

Comment:
The user doesn't actually supply a timeout (how long) but an actual point in time (indirectly may be timeout)

Reply:
Also we can say that when propagating the deadline from client to server what is actually sent is a timeout value so the clocks in the respective machines don't need to be synchronized i.e. the deadline feature does not depend on clocks being synchronized even if the user provides an actual point in time as the deadline

Reply:
Yes, I had it flipped here.

I added a paragraph in the deadline propagation section that talks about the deadline to timeout conversion approach.